### PR TITLE
Add May 2023 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--1950.org.readthedocs.build/

### CODE REVIEW

**Comment by plaindocs** [comment]
Oh meetup script, Cooool! I never dusted off the one I had somewhere when they changed the API access.

File: docs/blog/newsletter-may-2023.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.. post:: May 04, 2023
+   :tags: newsletter
+
+####################################
+Write the Docs Newsletter – May 2023
+####################################
+
+Aloha, documentarians! I hope any showers you may have experienced in April are paying off in flowers. There's lots blossoming within Write the Docs.
+
+The main thing going on this week is the upcoming `Portland conference </conf/portland/2023/>`__. Wishing all of you who will make it to Portand safe travels, and if you can't make it there, you can follow along with the main talks through the livestream.
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2023.rst]:
```suggestion
The main thing going on this week is the `Portland conference </conf/portland/2023/>`__. Wishing all of you who will make it to Portand safe travels, and if you can't make it there, you can follow along with the main talks through the livestream.
```

IMO if it's this weekit's not really upcoming? :man_shrugging:

Comment (plaindocs) [reply docs/blog/newsletter-may-2023.rst]: Maybe link to https://www.writethedocs.org/conf/portland/2023/livestream/ ?

**Review by plaindocs — APPROVED** [review]
:+1: neat!

Can we have some super minimal docs on how to run the script (auth?) at the top of it or elsewhere?

**Comment by CollierCZ** [comment]
I added a minimal explanation for the script. And there are at least some comments interspersed. Hope it's clear enough?

**Pull request merged**